packages/teams-js/test/public/media.spec.ts

```diff
@@ -439,6 +440,133 @@ describe('media', () => {
           } as DOMMessageEvent);
         });
       });
+
+      it('videoController notifyEventToHost should fail in default version of platform', async () => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
+          mobilePlatformMock.setClientSupportedSDKVersion(originalDefaultPlatformVersion);
+          const stopRes = new media.VideoController().stop();
+          stopRes.catch(error => {
+            expect(error).not.toBeNull();
+            expect(error.errorCode).toBe(ErrorCode.OLD_PLATFORM);
+          });
+        });
+      });
+
+      it('videoController notifyEventToHost is handled successfully', async () => {
```

Comment:
Junit doesn't test anything. plese use await with async before expect so the test would wait for the promise to be resolved.

Reply:
Opening a bug item and will link to it in our teams git magic channel, thanks!